docs/structured-streaming-programming-guide.md

```diff
@@ -1126,6 +1126,12 @@ aggDF \
 </div>
 </div>
 
+## Advanced Sources
```

Comment:
Well there it was explicitly categories as inbuilt sources and advanced sources before this section. Here that is not there. So the term "advanced" does not make sense.